services/snapstore/snapstore-version.service.js

```diff
@@ -0,0 +1,54 @@
+import Joi from 'joi'
+import { BaseJsonService, pathParams } from '../index.js'
+
+const versionSchema = Joi.object({
+  'channel-map': Joi.array().items(
+    Joi.object({
+      version: Joi.string().required(),
+    }),
+  ),
+}).required()
+
+export default class SnapstoreVersion extends BaseJsonService {
+  static category = 'version'
+
+  static route = {
+    base: 'snapstore/v',
+    pattern: ':package',
+  }
+
+  static defaultBadgeData = { label: 'snapcraft', namedLogo: 'snapcraft' }
```

Comment:
Oh, also can we ditch the `namedLogo`. Logos should be off by default unless it is a social badge. Thanks

Reply:
No problem, Fixed with https://github.com/badges/shields/pull/9976/commits/992569b1fba27ae15c8732c7ec8ad65b122a1eff